tsconfig.json

```diff
@@ -22,5 +22,36 @@
         "entryPoints": ["src/"],
         "entryPointStrategy": "expand",
         "out": "docs",
+        "cname": "chessbots.cometrobotics.org",
+        "searchInComments": true,
+        "navigationLinks": {
+            "GitHub Repository": "https://github.com/Comet-Robotics/chessBot",
+        },
+        "name": "Chess Bots",
+        "exclude": "**/node_modules/**/*",
+        "pluginPages": {
+            "source": "documentation",
```

Comment:
Do we officially want both documentation and docs folders? I was thinking they just go into the docs folder.

Reply:
Just docs :)